game/src/main/java/net/demilich/metastone/game/utils/Attribute.java

```diff
@@ -883,6 +878,31 @@
 	 */
 	ATTACK_MULTIPLIER,
 
+	/*
+	 * Will block an entity from receiving game event triggers
+	 */
+	CANT_GAIN_ENCHANTMENTS,
+
+	/*
+	 * Signifies that an enchantment should treat itself as the source for triggers rather than the source of the event
+	 */
+	USE_AS_SOURCE,
+
+	/*
+	 * Indicates this cards was the most recently shuffled into a deck
+	 */
+	LAST_SHUFFLED,
+
+	/*
+	 * Indicates a character shouldn't naturally lose the FROZEN attribute
+	 */
+	DONT_UNFREEZE,
+
+	/*
+	 * Indicates a character can't be destroyed
+	 */
+	CANT_BE_DESTROYED,
+
```

Comment:
Basically all the effects except DONT_UNFREEZE are too glitchy to actually do. Their cards should be staging'd